content/python/concepts/functions/terms/isinstance/isinstance.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'isinstance()'
+Description: 'Returns `True` if object is the specified type. Otherwise function will return `False`.'
+Subjects:
+  - 'Computer Science'
+  - 'Python'
+  - 'Engineering'
+  - 'Programming'
+  - 'Web Dev'
+  - 'Data Science'
+  - 'Data Analytics'
+Tags:
+  - 'Python'
+  - 'Functions'
+  - 'Built-in'
+  - 'Boolean'
+  - 'Objects'
+  - 'Types'
+  - 'TypeError'
+  - 'Parameters'
+  - 'Class'
+CatalogContent:
+  - 'python'
+  - 'functions/isinstance'
+---
+
+<h1>isinstance()</h1>
+<strong>Python function that evaluates if a value type equates to True</strong><br>
+
+## Description
+The isinstance() function determines whether one object is a designated value type. If it is, the function will return `True`, otherwise the function will return `False`.
```

Comment:
```suggestion
The **`isinstance()`** function determines whether one object is a designated value type. If it is, the function will return `True`, otherwise the function will return `False`.
```